src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2362,16 +2367,16 @@ module Types = struct
     end
 
     module SendTestZkappInput = struct
-      type input = Mina_base.Parties.t
+      type input = Mina_base.Zkapp_command.t
 
       let arg_typ =
-        scalar "SendTestZkappInput" ~doc:"Parties for a test zkApp"
+        scalar "SendTestZkappInput" ~doc:"Zkapp_command for a test zkApp"
```

Comment:
nit: would be nicer to write "Zkapp command" here, without underscore (or "ZkApp command", or "zkApp command"??)